### PR TITLE
[editor] Initialize `KeyboardManager`-instances lazily

### PR DESCRIPTION
**Pull request by Snuffleupagus**
As far as I can tell there's no particular reason for initializing `KeyboardManager`-instances eagerly, since the user may never use editing, and we can easily do this lazily instead by utilizing shadowed getters.

### CODE REVIEW

File: src/display/editor/freetext.js [hunk 1]
```diff
@@ -58,12 +59,15 @@ class FreeTextEditor extends AnnotationEditor {
...
   static _defaultFontSize = 10;
 
-  static _keyboardManager = new KeyboardManager([
-    [
-      ["ctrl+Enter", "mac+meta+Enter", "Escape", "mac+Escape"],
-      FreeTextEditor.prototype.commitOrRemove,
-    ],
-  ]);
+  static get _keyboardManager() {
+    const km = new KeyboardManager([
```
Comment (calixteman) [inline src/display/editor/freetext.js:63]: nit: I guess we can survive without having the `km` variable.

**Review by calixteman — APPROVED** [review]
LGTM. Thank you.

File: src/display/editor/freetext.js (hunk 1 above)
Comment (Snuffleupagus) [reply src/display/editor/freetext.js]: Sure, that works for me!

**Pull request merged**